externs/browser/w3c_permissions.js

```diff
@@ -23,10 +23,14 @@
 
 
 /**
- * @typedef {{name: PermissionName}}
+ * @record
```

Comment:
Just FYI,

changing this to no longer be a typedef exposed the fact that `Permissions.prototype.query` is declared incorrectly below. It's argument should be of type `!PermissionDescriptor`.

This broke some code within Google.
I've already applied the fix and am testing again to see what other problems there are.

Reply:
Thanks, I will apply the fix too.